client/src/main/java/io/pravega/client/control/impl/Controller.java

```diff
@@ -364,6 +365,16 @@
      */
     CompletableFuture<Transaction.Status> checkTransactionStatus(final Stream stream, final UUID txId);
 
+    /**
+     * Get list of TransactionInfo for the Stream having status COMMITTED/ABORTED from most recent batch.
+     * TransactionInfo contains unique transactionId, status of transaction and stream.
+     * This api can return maximum 200 records.
```

Comment:
Where this constant comes from? Shouldn't this be configurable?

Reply:
As per our implementation, a batch can have maximum 200 records. If no of records are more than 200 then new batch will be created and entries will be saved in new completedTransactionsBatch-batch table. This API is fetching entries from recent batch only, so it can return max 200 records.